opcua.cpp

```diff
@@ -719,6 +770,95 @@ void OPCUA::clearSubscription()
 	m_subscriptions.clear();
 }
 
+/**
+ * Returns whether Node is to be subscribed to, as per filtering config
+ */
+bool OPCUA::checkFiltering(std::string browseName, OpcUa_NodeClass nodeClass, bool isDirectlySubscribed)
+{
+	// if filtering is disabled, always subscribe to given node
+	if(!getFilterEnabled())
+	{
+		Logger::getLogger()->info("%s:%d: FILTERING IS DISABLED", __FUNCTION__, __LINE__);
```

Comment:
At debug level, we have dataChange related logs, which can be excessive and this comparatively important information could get lost in the deluge of those messages. Hence, this is kept at a slightly higher level. This filtering logic is invoked only at startup and reconf times. Let me know if you still think otherwise.

Reply:
I think we can't leave just 1 single (debug) log level for developer. I think, INFO and DEBUG levels belong to developer. Default debug level for our C++ services is WARNING, so INFO and DEBUG level logs are typically collected when trying to understand some situation/problem better or requested by developer.

Reply:
And those logs without nodeid or browsename made sense when seen in the sequence they appeared in syslog. But, I have moved all logs in checkFiltering() to debug level.